integration/test/dashboard_test.go

```diff
@@ -77,7 +77,7 @@ func (suite *DashboardImportTestSuite) TestCModeDashboardCount() {
 		"NetApp Detail: Network with NVMe/FC", "NetApp Detail: Node - Details",
```

Comment:
let's make this easier to read and diff by putting on seperate lines
```go
expectedName := []string{
		"Harvest Metadata",
		"NetApp Detail: Aggregate",
		"NetApp Detail: Cluster",
		"NetApp Detail: Disk",
		"NetApp Detail: LUN",
		"NetApp Detail: Network  - Details",
		"NetApp Detail: Network with NVMe/FC",
		"NetApp Detail: Node - Details",
		"NetApp Detail: Shelf",
		"NetApp Detail: SnapMirror",
		"NetApp Detail: SVM - Details",
		"NetApp Detail: Volume - Details",
		"NetApp Detail: MetroCluster",
		"NetApp Detail: Data Protection",
		"NetApp Detail: Data Protection SnapMirror",
        "NetApp Detail: Security",
	}
```

Reply:
done